python/paddle/incubate/optimizer/lookahead.py

```diff
@@ -112,9 +122,21 @@ class LookAhead(Optimizer):
 
     """
 
+    inner_optimizer: Optimizer
+    alpha: float
+    k: int
+    type: str
+    helper: LayerHelperBase | None
```

Comment:
```suggestion
    helper: LayerHelper
```

没必要用 `LayerHelperBase` 吧？